enos/enos-qualities.hcl

```diff
@@ -602,6 +602,11 @@ quality "vault_secrets_kv_write" {
   description = "Vault kv secrets engine data is writable"
 }
 
+
+quality "vault_secret_engine_config_update" {
```

Comment:
`vault_secrets_ldap_write_config` ? to match the existing style and describe which config we're updating?